templates/locales/en.yml

```diff
@@ -2039,12 +2039,16 @@ en:
         upload_path_not_set: "`upload_path` must be set for the shell provisioner."
 
       ansible:
+        cannot_detect: |-
+          Vagrant does not support detecting whether Ansible is installed
+          for the guest OS running in the machine. Vagrant will assume it is
+          installed and attempt to continue.
```

Comment:
If I understand well, you mean give some hints/invitation to help in improving Vagrant on this point.
By chance, do you know if there is already some similar message in Vagrant or maybe another Hashicorp project (just to make things look great and coherent ;-)